src/rust/ide/lib/view/src/graph_editor/builtin/visualization/native/documentation/style.css

```diff
@@ -0,0 +1,429 @@
+/*
+ * TODO TODO TODO TODO TODO TODO TODO TODO TODO TODO TODO TODO TODO TODO TODO
+ *      This file is undergoing drastic changes.
```

Comment:
I asked in previous review to explain what "drastic changes are". This description is very enigmatic.

Reply:
![image](https://user-images.githubusercontent.com/1623053/90576691-31479780-e1bf-11ea-99c6-44b8a97c4e7c.png)